.ci_support/osx_64_.yaml

```diff
@@ -1,5 +1,5 @@
 MACOSX_DEPLOYMENT_TARGET:
-- '10.9'
+- '10.13'
```

Comment:
I was going to take this green PR, but see this change was included. @zklaus , does this concern you?

Reply:
I am ok with it. To be honest, not being an apple guy myself, I am not completely sure what it means, but I think it came with the recent `{{ stdlib('c') }}` migration and might actually just be making things consistent.

Reply:
Hm. Looking in meta yaml, there now is a duplication of that dependency. Let's fix that.